### ISSUE TITLE
Investigate where time is spent on dev & reference machine

### ISSUE THREAD

**Issue by wdanilo** [enso-org/enso#4285]
This task is automatically imported from the old Task Issue Board and it was originally created by Michael Mauderer.
Original issue is [here](https://www.pivotaltracker.com/story/show/181651660).
<hr />

## BEFORE YOU START, REMEMBER TO WRITE A GOOD TITLE
The title should be short, descriptive.

## Description
Why is it needed? Does it help the team go faster or is it a dependency that could cause problems in the codebase if it’s not done?

## Resources
Including instructions, additional context, or other assets that help execute the chore. **Pivotal Tracker does not allow you to upload media as part of the description. Use the first comment to attach files instead.** Remove this section if not needed.

## Comments:
**Michael Mauderer** reports a new **STANDUP** for yesterday (2022-03-22):

**Progress:** Analysis if profiling data. It should be finished by 2022-03-24.

**Next Day:** Next day I will be working on the [#same task](https://www.pivotaltracker.com/story/show/181651660). More looking into profiling data and gather some more data on a weaker reference machine.  Update FRP Profiling PR after merge of Adams component changes. (Enso Bot - Mar 23, 2022)
<hr />
**Michael Mauderer** reports a new **STANDUP** for yesterday (2022-03-23):

**Progress:** Set up two older windows machines (one without dGPU) for profiling purposes. Ran into some issues there, which took some extra time. Collected data. It should be finished by 2022-03-24.

**Next Day:** Next day I will be working on the [#same task](https://www.pivotaltracker.com/story/show/181651660). More looking into profiling data and discussions about next steps / planning. (Enso Bot - Mar 24, 2022)
<hr />
**Michael Mauderer** reports a new **🔴 DELAY** for yesterday (2022-04-05):

**Summary:** There is **14 days** delay in implementation of the [Investigate where time is spent on dev & reference machine (#181651660) task](https://www.pivotaltracker.com/story/show/181651660).
It **will cause 0 days delay** for the delivery of this weekly plan.

I accidentally reported stand-ups and delays for the wrong task. I';m switching to the correct task now, but need to update the end time.

**Delay Cause:** Confused developer.

**Possible solutions:** Keeping better track on the task that standups are reported for. A web form showing the title of the task that is selected might also avoid this type of error. (Enso Bot - Apr 6, 2022)
<hr />
**Michael Mauderer** reports a new **STANDUP** for yesterday (2022-04-05):

**Progress:** : Update visualisation to show time async tasks spend sleeping, which greatly improves how easy it is to sot what the IDE is waiting on in the background. Started removing some code that is no longer needed and cleaned up the profiling and visualisations. Taking a bit longer than expected. It should be finished by 2022-04-06.

**Next Day:** Next day I will be working on the [#same task](https://www.pivotaltracker.com/story/show/181651660). Finish the PR. (Enso Bot - Apr 6, 2022)
<hr />
**Michael Mauderer** reports a new **STANDUP** for yesterday (2022-04-06):

**Progress:** : Finished the PR with the changes mad in the course of investigating the back-end wait times. Did PR review for shader compile PR. It should be finished by 2022-04-06.

**Next Day:** Next day I will be working on the [##181151201 task](https://www.pivotaltracker.com/story/show/181151201). Implement review feedback (if it is there). Integrate EnsoGL stats with profiling framework (should be quick). Then start looking at whether we can already get some manual data on UI interactions like collapsing nodes. (Enso Bot - Apr 7, 2022)
<hr />
